src/Dapper.Azure.WebJobs.Extensions.SqlServer/Dapper.Azure.WebJobs.Extensions.SqlServer.csproj

```diff
@@ -2,14 +2,9 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
-    <PackageId>Dapper.Azure.WebJobs.Extensions.SqlServer</PackageId>
-    <Title>SQL binding for azure functions</Title>
-    <Authors>Kristian Gundry Nielsen</Authors>
-    <PackageDescription>SQL binding for azure functions using Dapper, currently only working with MS SQL Server</PackageDescription>
-    <PackageTags>azure-function sql dapper azure</PackageTags>
-    <Copyright>Copyright (c) 2019 Kristian Gundry Nielsen</Copyright>
-    <RepositoryUrl>https://github.com/kristianGNI/dapper-azure-function-binding</RepositoryUrl>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>    
+    <NuspecFile>..\..\nuget\Dapper.Azure.WebJobs.Extensions.SqlServer.nuspec</NuspecFile>
```

Comment:
Would it make sense to have the nuspec file together with the library instead of <root>/nuget ?

Reply:
yeah, agree makes more sense

Reply:
i'll move it around